modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -655,6 +655,29 @@ const register = (editor: Editor) => {
     processor: 'string'
   });
 
+  registerOption('text_patterns', {
+    processor: (value) => {
+      if (Type.isArrayOf(value, Type.isObject) || value === false) {
+        return { value, valid: true };
```

Comment:
Maybe normalize this to an empty array no patterns should be the same as false. I've been thinking that we could do that for a lot of things setting strings to false could be internally an empty string. Then the types becomes less messy internally at least.

Reply:
Done